documentation/source/contributing/contribute-code.rst

```diff
@@ -5,3 +5,38 @@ Contribute code
 - Try to find a `first good issue <https://github.com/inexorgame/vulkan-renderer/issues>`__ in our issue tracker.
 - Open a `pull request <https://github.com/inexorgame/vulkan-renderer/pulls>`__.
 - We want to keep commit as small in size as possible.
+
+Signing commits
+---------------
+
+- We encourage you to sign your commits, although this is not a strict requirement from our side.
+- You can find help on how to sign commits in `GitHub's docs <https://docs.github.com/en/github/authenticating-to-github/signing-commits>`__.
+
+Commit naming convention
+------------------------
+
+The commit naming convention will be checked by our continuous integration. In order to be valid, the followin rules must all be fulfilled:
+
+- The commit message must begin with a category in square brackets which describes the code part that was changed
+- The commit category it must not be empty 
+- A commit category must contain at least two characters. Allowed characters are: letters ``a-z``, numbers ``0-9``, and ``-``. For example ``[gpu-texture]``
+- If multiple code parts are affected, separate the categories using ``|``, for example ``[gpu-texture|gpu-info]``
+- If two or more code parts are affected, use ``[*]`` as category
```

Comment:
These two statements together are a little contradictory/confusing, maybe it should be `If three or more code parts are affected...`?

Reply:
Done.